app/TuistApp/Resources/Assets.xcassets/MenuBarIcon.imageset/Contents.json

```diff
@@ -19,5 +19,8 @@
   "info" : {
     "author" : "xcode",
     "version" : 1
+  },
+  "properties" : {
+    "template-rendering-intent" : "template"
```

Comment:
Curious why this was added?

Reply:
When the menu bar background is light-colored, the icon becomes difficult to recognize.

before:
<img src="https://github.com/user-attachments/assets/26ea42dd-fb57-4682-acfe-d4cac826f76d" alt="Description" width="160"/>
after:
<img src="https://github.com/user-attachments/assets/e8253883-8708-4ceb-9be1-4151448f6c5a" alt="Description" width="160"/>

Reply:
Ahh, I see, thanks!